.devcontainer/devcontainer.json

```diff
@@ -1,19 +1,79 @@
 {
-    "dockerComposeFile": ["../development/docker-compose.yml", "../development/docker-compose.debug.yml"],
+    "name": "Nautobot Dev Container",
+    "dockerComposeFile": [
+        "../development/docker-compose.yml",
+        "../development/docker-compose.postgres.yml",
+        "../development/docker-compose.dev.yml",
+        "../development/docker-compose.override.yml"
```

Comment:
Does this make it so that a `docker-compose.override.yml` *must* be provided by the user? Or will this still work if no such file is provided?

Reply:
It will still work without this file. I use custom `docker-compose.override.yml` and `override.env` files to automatically bootstrap Nautobot with a user and token, as explained [here](https://docs.nautobot.com/projects/core/en/stable/development/core/docker-compose-advanced-use-cases/#automatically-creating-a-superuser). I can remove them if you think they are unnecessary.

Reply:
Is there any reason why you don't want superuser information in `dev.env`? For development purpose, I find incredibly useful to automatically start Nautobot with a superuser, so I don't have to worry recreating it every time my setup is destroyed/rebuilt.

Reply:
I cloned a clean copy of this branch and the devcontainer fails to start with this error:

```
[752829 ms] Start: Run: docker ps -q -a --filter label=devcontainer.local_folder=/home/gary/github/nautobot_devcontainer --filter label=devcontainer.config_file=/home/gary/github/nautobot_devcontainer/.devcontainer/devcontainer.json
[752857 ms] Start: Run: docker ps -q -a --filter label=devcontainer.local_folder=/home/gary/github/nautobot_devcontainer
[752882 ms] Resolving Feature dependencies for 'ghcr.io/devcontainers/features/docker-outside-of-docker:1'...
[752882 ms] * Processing feature: ghcr.io/devcontainers/features/docker-outside-of-docker:1
[753107 ms] Start: Run: docker-credential-secret get
[753606 ms] * Processing feature: ghcr.io/devcontainers/features/common-utils
[753853 ms] Soft-dependency 'ghcr.io/devcontainers/features/common-utils' is not required.  Removing from installation order...
[753856 ms] * Fetching feature: docker-outside-of-docker_0_oci
[754196 ms] * Fetched feature: docker-outside-of-docker_0_oci version 1.3.1
[754200 ms] Start: Run: docker-compose version --short
[754206 ms] Start: Run: docker compose version --short
[754318 ms] Docker Compose version: 2.21.0
[754318 ms] Start: Run: docker compose -f /home/gary/github/nautobot_devcontainer/development/docker-compose.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.postgres.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.dev.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.override.yml --profile * config
[754433 ms] 
[754434 ms] stat /home/gary/github/nautobot_devcontainer/development/docker-compose.override.yml: no such file or directory

[754434 ms] Exit code 14
[754434 ms] Error: Command failed: docker compose -f /home/gary/github/nautobot_devcontainer/development/docker-compose.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.postgres.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.dev.yml -f /home/gary/github/nautobot_devcontainer/development/docker-compose.override.yml --profile * config
    at Wg (/home/gary/.vscode/extensions/ms-vscode-remote.remote-containers-0.327.0/dist/spec-node/devContainersSpecCLI.js:460:889)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async TJ (/home/gary/.vscode/extensions/ms-vscode-remote.remote-containers-0.327.0/dist/spec-node/devContainersSpecCLI.js:390:23529)
    at async _eA (/home/gary/.vscode/extensions/ms-vscode-remote.remote-containers-0.327.0/dist/spec-node/devContainersSpecCLI.js:612:35354)
[754442 ms] Exit code 1
```

so it looks like the `docker-compose.override.yml` is required

Reply:
@gsnider2195 Sorry, I was not clear enough. The devcontainer will start without the`docker-compose.override.yml` file, but you need to remove it from `.devcontainer/devcontainer.json`.